_data/changelogs/component-collection.yml

```diff
@@ -2,6 +2,13 @@ title: Collection
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed deprecated xlink references.
+    summaryAdditional: All deprecated xlink:href references have been replaced with href.
```

Comment:
**Suggestion:** The second `deprecated` feels slightly unnecessary here. What do you think about removing it?

```suggestion
    summary: Removed deprecated xlink references.
    summaryAdditional: All xlink:href references have been replaced with href.
```